src/components/StatusCard/Components/Compact/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import clsx from 'clsx';
+import Typography from '../../../Typography';
+import { Props } from './types';
+import styles from './styles.scss';
+
+
+const CompactStatusCard = ({ className, color, header, body, disabled, fill, ...otherProps }: Props) => {
+
+    const compactClasses = clsx(
+        styles.wrapper,
+        styles[`color--${color}`],
+        disabled && styles.isDisabled,
```

Comment:
Should probably be written as `{ [styles. isDisabled]: disabled }` for readability.